src/assets/events.json

```diff
@@ -0,0 +1,29 @@
+[
+      {
+          "date": "Date",
```

Comment:
Why do some of these just say date?

Reply:
These should match the dates on the timeline in the powerpoint.

Reply:
Some of the events had no dates just descriptions @EVogel1999 @pzalep1 
I will ask at standup for the dates